### ISSUE TITLE
[Feature request] log following

### ISSUE THREAD

**Issue by n0n0x** [deis/workflow-cli#102]
It could be really useful to have a similar approach as `tail -f` for `deis logs`.

**Comment by bacongobbler** [comment]
related: deis/deis#465

**Comment by JeanMertz** [comment]
Would be nice to see this feature implemented 👍 

In the mean time, something like `watch 'deis logs | tail'` does the trick, sort of.

**Comment by bacongobbler** [comment]
@JeanMertz `kubectl logs -f` is another workaround. :)

**Comment by JeanMertz** [comment]
@bacongobbler very true :) but Deis logs include deployment logs as well, and the deis CLI doesn't require access to the kubernetes cluster, so that's less of a security risk.

**Comment by bacongobbler** [comment]
Indeed, I know that's just the best workaround for now. I know v1 users were happy to use `fleetctl journal` for their use case, so it's one alternative.

**Comment by dmcnaught** [comment]
+1

**Comment by Cryptophobia** [comment]
This issue was moved to teamhephy/workflow-cli#27
